features/step_definitions/machine.rb

```diff
@@ -57,23 +57,24 @@
 
 Then(/^admin ensures machine number is restored after scenario$/) do
   ensure_admin_tagged
-  cb.all_machines = BushSlicer::Machine.list(user: admin, project: project("openshift-machine-api"))
-  machine_orig_num = cb.all_machines.length
+  machine_names_orig = BushSlicer::Machine.list(user: admin, project: project("openshift-machine-api")).map { | m | m.name }
```

Comment:
a little shortening can be done here with:
```
BushSlicer::Machine.list(user: admin, project: project("openshift-machine-api")).map(&:name)
```
This is a shorter way to tell call `#name` method of the object. And only works when parameters are not needed.